src/main/scala-2.12/org/ergoplatform/appkit/ergotool/dex/CreateSellOrderCmd.scala

```diff
@@ -86,22 +86,40 @@ case class CreateSellOrderCmd(toolConf: ErgoToolConfig,
 }
 
 object CreateSellOrderCmd extends CmdDescriptor(
-  name = "dex:SellOrder", cmdParamSyntax = "<wallet file> <sellerAddr> <ergPrice> <tokenId> <tokenAmount> <dexFee>",
-  description = "put a token seller order with given <tokenId> and <tokenAmount> for sale at given <ergPrice> price with <dexFee> as a reward for anyone who matches this order with buyer, with <sellerAddr> to be used for withdrawal \n " +
-    "with the given <wallet file> to sign transaction (requests storage password)") {
+  name = "dex:SellOrder", cmdParamSyntax = "<storageFile> <sellerAddr> <tokenPrice> <tokenId> <tokenAmount> <dexFee>",
+  description = "put a token seller order with given <tokenId> and <tokenAmount> for sale at given <tokenPrice> price with <dexFee> as a reward for anyone who matches this order with buyer, with <sellerAddr> to be used for withdrawal \n " +
+    "with the given <storageFile> to sign transaction (requests storage password)") {
+
+  override val parameters: Seq[CmdParameter] = Array(
+    CmdParameter("storageFile", FilePType,
+      "storage with secret key of the sender"),
+    CmdParameter("storagePass", SecretStringPType,
+      "password to access sender secret key in the storage", None,
+      Some(ctx => ctx.console.readPassword("Storage password>"))),
+    CmdParameter("sellerAddr", AddressPType,
+      "address of the seller"),
```

Comment:
Remove sellerAddr

Reply:
I planned to do this in my cancel order command branch - #18.